src/report.js

```diff
@@ -5,29 +5,53 @@ const api = require('./api');
 const payloads = require('./payloads');
 
 /*
- *  Send report creation confirmation via
- *  chat.postMessage to #tsar_committee
+ *  Send report via chat.postMessage to #tsar_committee
  */
-const sendConfirmation = async(report) => {
+const sendReport = async(report) => {
 
-  let message = payloads.confirmation({
+  let message = payloads.report({
     channel_id: process.env.ANASTASIA_SLACK_CHANNEL,
     report: report.report,
     user: report.user_id,
   });
 
   let result = await api.callAPIMethod('chat.postMessage', message);
-  debug('sendConfirmation: %o', result);
+  debug('sendReport: %o', result);
+};
+
+/*
+ *  Send sign-off via chat.postMessage to #tsar_committee
+ */
+const sendSignOff = async(signoff) => {
+
+  let message = payloads.signoff({
+    channel_id: process.env.ANASTASIA_SLACK_CHANNEL,
+    user: signoff.user_id,
+  });
+
+  let result = await api.callAPIMethod('chat.postMessage', message);
+  debug('sendSignOff: %o', result);
 };
 
 // Create report. Call users.info to get the username
 // from their user ID
 const create = async(userId, view) => {
 
+  let user_info = await api.callAPIMethod('users.info', { user: userId });
+  let user_name = user_info.user.real_name;
+
   let user_report = view.state.values.notes_block.report_text.value;
+  let send_to_lw = view.state.values.lw_block.lw.selected_options.length > 0;
+
+  if (send_to_lw) {
+    let lightwood_post = `<@${process.env.ANASTASIA_LIGHTWOOD_SLACK_ID}> post`;
```

Comment:
You can't embed Lightwood's username in a text block like this if you want Lightwood to understand it.  You need to use a user block, as if it were a message coming from a client:

https://api.slack.com/reference/block-kit/blocks#user-element-type

Reply:
No, I think this is wrong now...